src/remote-desktop.c

```diff
@@ -1374,6 +1476,8 @@ remote_desktop_iface_init (XdpDbusRemoteDesktopIface *iface)
   iface->handle_notify_touch_down = handle_notify_touch_down;
   iface->handle_notify_touch_motion = handle_notify_touch_motion;
   iface->handle_notify_touch_up = handle_notify_touch_up;
+
```

Comment:
Please remove this newline

Reply:
Let's not block on this

Reply:
ftr, newline is there to maintain existing grouping, `create/select/start` are in one group and the various `notify` functions in another. Happy to remove it if you want me to though.

Reply:
Nah, it's all good.